packages/core-modules/contracts/test/RouterMock.sol

```diff
@@ -0,0 +1,50 @@
+//SPDX-License-Identifier: MIT
```

Comment:
Shouldn't this be in the 'mocks' folder. I thought we'd renamed it.

Reply:
Yes we did... I had a previous version of the folders :( Moved